src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task {
+    protected String from;
+    protected String to;
```

Comment:
I like the simplicity of the names of the time variables. However, it may be abit more clearer if the variables were more specific such as the following:

```suggestion
    protected String fromDateTime;
    protected String toDateTime;
```